node_modules/col-whiteboards/tests/util.js

```diff
@@ -92,6 +93,30 @@ var assertWhiteboard = module.exports.assertWhiteboard = function(whiteboard, op
 var assertCreateWhiteboard = module.exports.assertCreateWhiteboard = function(client, course, title, members, callback) {
   UsersTestUtil.assertGetMe(client, course, null, function(me) {
 
+    // Collect any emails that are sent out when creating the whiteboard
+    var emails = [];
+    function emailListener(email, user, course) {
+      emails.push({
+        'email': email,
+        'user': user,
+        'course': course
+      });
+    };
+    EmailUtil.on(EmailUtil.EVENT_NAMES.EMAIL_SENT, emailListener);
+
+    // Because of the asynchronous process of sending out emails, we only return to the caller when
+    // both the REST request finishes and the emails have been sent out
+    var _whiteboard = null;
+    var done = _.after(2, function() {
```

Comment:
Shouldn't this be the number of members (minus myself) rather than a hardcoded 2 ?